clang/include/clang/Format/Format.h

```diff
@@ -5274,9 +5274,17 @@ struct FormatStyle {
 
   friend std::error_code
   parseConfiguration(llvm::MemoryBufferRef Config, FormatStyle *Style,
+                     const std::vector<std::string> &StyleSearchPaths,
```

Comment:
by putting your argument 3rd you made every instance of this function need to change..

you could have added it last and used a default argument, or made an overloaded function and a wrapper for the old one to call the new with the empty argument this would cut down the flux

Reply:
Yes, I can certainly do that if that is preferred. And, if I'm to do this as multiple PRs as several have requested, then a default param (or an overload) is needed. FWIW, though, I did this intentionally, to flush out all callers of lib/format: once this feature is in and used by actual users, this needs to be pushed through all the tools that leverage lib/format, or else those tools will choke on peoples' valid .clang-format files.